db.js

```diff
@@ -168,3 +175,27 @@ export const populateUserCache = async (cache = userCache) => {
 	}
 };
 
+
+
+export const insertTimezone = async ( _id, timezone = null , dbConnection = connectDB) => {
+	try {
+		const db = await dbConnection();
+		await db.updateOne({_id}, {$set: {timezone}})
```

Comment:
I think the way this is written right now, there's a chance that a users timezone prop could be `null`, do we want that or should the function check to make sure there's a value for timezone?

Reply:
so, this function is actually being used in 3 places `--update`, `--sub` and `--unsub` (talk about reusing code). only when used in `--unsub` it won't pass the timezone and will define the value as of that prop as `null`. so yes, this is expected behavior.